gamemaster.php

```diff
@@ -220,6 +219,23 @@
 
 	try
 	{
+
+		$split = explode(',', $Game->datetxt());
+		$season = $split[0];
+		$currentYear = intval($split[1] ?? 1901);
+		$endYear = $Game->endYear;
+
+		if ($currentYear == $endYear && $season == "Autumn" && ($Game->phase == "Builds" || $Game->phase == "Retreats")) {

```

Comment:
Is season "Autumn" or "Winter" during builds?

Also, it's possible for both retreats and builds to be skipped (no retreats, no builds), which would miss the end date.

Add `|| $currentYear > $endYear`?

Reply:
oh good point